pkg/crc/preflight/preflight_checks_network_linux.go

```diff
@@ -15,6 +15,13 @@ import (
 )
 
 var nmPreflightChecks = [...]Check{
+	{
+		configKeySuffix:  "check-systemd-networkd-not-running",
```

Comment:
Shouldn't the `configKeySuffix` should be `check-systemd-networkd-running` as per description and then user can skip this check with config set?

Reply:
yeah should work this way too, and it's a less awkward name, thanks!